hydromt_wflow/naming.py

```diff
@@ -787,75 +787,75 @@
         "wflow_v0": "lateral.river.rhos",
         "wflow_v1": "sediment__particle_density",
     },
-    "D50_Engelund": {
+    "river_sediment_d50": {
         "wflow_v0": "lateral.river.d50engelund",
         "wflow_v1": "river_sediment__d50_diameter",
```

Comment:
Maybe this one was forgotten in the other PR? It's been deleted in v1

```suggestion
        "wflow_v1": None,
```